LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
It's unnecessary. The license fields on the package manifest accepts any SPDX identifier, so "MIT" is enough there.